test/regress/cli/run_regression.py

```diff
@@ -174,11 +174,106 @@ def applies(self, benchmark_info):
     def run(self, benchmark_info):
         return super().run(
             benchmark_info._replace(
-                command_line_args=benchmark_info.command_line_args + ["--check-proofs", "--proof-granularity=theory-rewrite"]
+                command_line_args=benchmark_info.command_line_args +
+                ["--check-proofs", "--proof-granularity=theory-rewrite"]
             )
         )
 
 
+class LfscTester(Tester):
+    def applies(self, benchmark_info):
+        return (
+            benchmark_info.benchmark_ext != ".sy"
+            and (
+                "unsat" in benchmark_info.expected_output.split()
+                or "entailed" in benchmark_info.expected_output.split()
+            )
+            and "-i" not in benchmark_info.command_line_args
+            and "--incremental" not in benchmark_info.command_line_args
+            and "--no-produce-proofs" not in benchmark_info.command_line_args
+            and "--no-check-proofs" not in benchmark_info.command_line_args
+            and "--check-proofs" not in benchmark_info.command_line_args
+            and "--produce-unsat-cores" not in benchmark_info.command_line_args
+            and "--dump-unsat-cores" not in benchmark_info.command_line_args
+            and ":incremental true" not in benchmark_info.benchmark_content
+            and ":produce-proofs false" not in benchmark_info.benchmark_content
+            and ":check-proofs" not in benchmark_info.benchmark_content
+            and ":produce-unsat-cores true" not in benchmark_info.benchmark_content
+            and ":dump-unsat-cores true" not in benchmark_info.benchmark_content
+            and "(reset)" not in benchmark_info.benchmark_content
```

Comment:
I'd prefer not to have any of those conditions hard coded if we can avoid them (they are too brittle and it makes it harder to tell on which benchmarks the tester is actually being run). We have been trying to move to explicitly disabling testers instead.

Reply:
I reduced them to 2 conditions. What do you think of the new condition?